pkg/api/merge_request.go

```diff
@@ -52,6 +52,22 @@ var ListMRs = func(client *gitlab.Client, projectID interface{}, opts *gitlab.Li
 	return mrs, nil
 }
 
+var ListMRsWithAssignees = func(client *gitlab.Client, projectID interface{}, opts *gitlab.ListProjectMergeRequestsOptions, assigneeIds []int) ([]*gitlab.MergeRequest, error) {
+	if client == nil {
+		client = apiClient
+	}
+	mrs := make([]*gitlab.MergeRequest, 0)
+	for _, id := range assigneeIds {
+		opts.AssigneeID = gitlab.Int(id)
+		assingeMrs, _, err := client.MergeRequests.ListProjectMergeRequests(projectID, opts)
```

Comment:
Since we already have `ListMRs` function, why don't you just call that with `ListMRs(client, projectID, opts)`  instead of `client.MergeRequests.ListProjectMergeRequests(projectID, opts)`. 
I know this whole `api` package is a silly wrapper but I'll prefer to keep it that way in case we want to switch from the go-gitlab package

Reply:
Yeah, this makes sense. That is the same thing I am doing for users in `api` package. Changed this.

Reply:
Yeah, this makes sense. That is the same thing I am doing for users in `api` package. Changed this.